### PR TITLE
Fixes #20911 - Add welcome page for arf reports

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

File: app/views/arf_reports/welcome.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="blank-slate-pf">
+  <div class="blank-slate-pf-icon">
+    <%= icon_text("book", "", :kind => "fa") %>
+  </div>
+  <h1><%= _('Compliance Reports') %></h1>
+  <p>
+    <%= _("You don't seem to have any reports.") %></br>
```
Comment (ares) [inline app/views/arf_reports/welcome.html.erb]: Looks great, could you please expand a test a bit, e.g. to

"You don't seem to have any ARF report. ARF report is a summary of a single scan occurrence on a particular host for a given Compliance Policy."

**Comment by ares** [comment]
[test]

**Comment by shiramax** [comment]
@ares Done.

**Comment by xprazak2** [comment]
[test]

**Comment by iNecas** [comment]
We can probably merge this one, right?

**Comment by xprazak2** [comment]
Merging, thanks @shiramax!

**Pull request merged**